docs/using-cinder-csi-plugin.md

```diff
@@ -30,10 +30,101 @@ by using the result of the above command.
 
 ```kubectl -f manifests/cinder-csi-plugin create```
 
-### Example Nginx application
+This creates a set of cluster roles, cluster role bindings, and statefulsets etc to communicate with openstack(cinder).
+For detailed list of created objects, explore the yaml files in the directory.
+You should make sure following similar pods are ready before proceed:
+
+```
+NAME                                READY   STATUS    RESTARTS   AGE
+csi-attacher-cinderplugin-0         2/2     Running   0          29h
+csi-nodeplugin-cinderplugin-xxfh5   2/2     Running   0          46h
+csi-provisioner-cinderplugin-0      2/2     Running   0          46h
+```
+
+### Example Nginx application usage
+
+After perform above steps, you can try to create StorageClass, PersistentVolumeClaim and pod to consume it.
+Try following command:
 
 ```kubectl -f examples/cinder-csi-plugin/nginx.yaml create```
 
+You will get pvc which claims one volume from cinder 
+
+```
+root@ji1:~/go/src/k8s.io/cloud-provider-openstack/examples/cinder-csi-plugin$ kubectl get pvc
```

Comment:
I mean Instead of using this-
root@ji1:~/go/src/k8s.io/cloud-provider-openstack/examples/cinder-csi-plugin$ kubectl get pvc
It is more neat-
$ kubectl get pvc

Reply:
right ... I copied from my local env and didn't realize that.....